mod/wikirate_assets/set/all/wikirate_shared.rb

```diff
@@ -0,0 +1,48 @@
+NAV_MENUS = {
+  "Explore" => %w[Companies Topics Metrics Projects Groups Sources Changes],
+  "How To" => ["Add Data", "Use Data", "Organize Projects", "FAQ", "Report Issue"],
+  "About" => ["About Us", "Our Team", "Programs", "Publications", "News",
+              "Contact Us", "Donate"]
+}.freeze
+
+MENU_REFS = { "Groups" => "Research Groups",
+              "Changes" => ":recent",
+              "FAQ" => "Frequently Asked Questions",
+              "Report Issue" => "Tickets",
+              "Add Data" => "How to Add Data",
+              "Use Data" => "How to Use Data",
+              "Organize Projects" => "How to Organize Projects",
+              "Facebook" => "https://www.facebook.com/wikirate/",
+              "Twitter" => "https://twitter.com/WikiRate",
+              "Instagram" => "https://www.instagram.com/wikirate/",
+              "Telegram" => "https://t.me/WikiRate",
+              "Notice" => "Notice and Take Down Procedure",
+              "Legal" => "https://project.wikirate.org/Legal" }.freeze
+
+SECONDARY_MENUS = { "Legal" => ["Legal", "Privacy Policy", "Licensing", "Disclaimers",
+                                "Terms of Use", "Notice"],
+                    "Social" => %w[Facebook Twitter Instagram Telegram] }.freeze
+
+format do
+  def nav_menus
+    NAV_MENUS
+  end
```

Comment:
If your constants have to be overwritable you can use the setting api
```ruby
format do
  setting :nav_menus
  nav_menus NAV_MENUS
end
```

Reply:
Cool.  In this case it wasn't for overwriting; I just wanted a convenient way to pull the hash into haml templates.  I refer to it in three places, so the full constant name was unwieldy.